README.md

```diff
@@ -211,14 +226,14 @@ Or you can just add it to your `deps.edn` file as an alias:
 ```clojure
 {:deps {}
  :aliases
- {:clj-watson {:extra-deps {io.github.clj-holmes/clj-watson {:git/tag "v5.1.3" :git/sha "5812615"}}
-               :main-opts ["-m" "clj-watson.cli" "scan"]}}}
+ {:clj-watson {:replace-deps {io.github.clj-holmes/clj-watson {:git/tag "v5.1.3" :git/sha "5812615"}}
+               :main-opts ["-m" "clj-watson.cli"]}}}
```

Comment:
Since `scan` is a required argument, and there are no other possible commands to run, why put the burden on the user to supply it? Or is this intended to be more consistent with the tool usage which requires the user specify the `scan` function?

Reply:
Yeah, consistency and, could argue, future-proofing if we add more commands.

Reply:
Gonna assume you are good with this as is. Lemme know.